.github/workflows/ci.yml

```diff
@@ -38,17 +38,17 @@ jobs:
           restore-keys: ${{ runner.os }}-composer-
```

Comment:
@lfortunier i think the composer cache step is messing with the flex recipe install that should add the make/qualimetry.mk and make/test.mk.

Try to remove the steps "Get composer cache directory" and "Cache composer dependencies"